pkg/operator/controller/controller.go

```diff
@@ -752,6 +757,22 @@ func (r *ReconcileVMImportConfig) getOperatorArgs(cr *vmimportv1alpha1.VMImportC
 		}
 	}
 
+	// update operatorArgs with OS config map name and namespace if defined in vm-import-operator deployment
```

Comment:
Do you want to customize it in csv generator?

Reply:
I'll just add those env-vars definition, but providing a value for them should be done by a user if needed.